### PR TITLE
MRG: Remove double-backticks

### LINKED ISSUE #5118: Beautify some docs properties
There are some things in the auto-generated docs website that could be improved.

1. Sometimes, the listed attributes are enclosed in double backticks (e.g. https://martinos.org/mne/dev/generated/mne.preprocessing.Xdawn.html#mne.preprocessing.Xdawn, https://martinos.org/mne/dev/generated/mne.decoding.CSP.html#mne.decoding.CSP). These double backticks get rendered in the docs, which is probably not what we want.
2. If there is a references section, each listed reference is preceded by `(1, 2)` which should point to all occurrences in the text. However, the `2` entry is there even though a reference is only mentioned once in the text, and thus the link doesn't work.
3. I wonder if it's correct to use double backticks when referring to arguments in the description (see e.g. the CSP docs, where the `reg` parameter is referred to in the description). In the type description of the parameters, these are usually rendered italic, so maybe we should enclose only with single backticks to be consistent.

### PR DESCRIPTION
**Pull request by larsoner**
Closes #5118

So far it looks locally like the double-backtics are no longer necessary on `var_` names in `Attributes` sections. We'll see if CircleCI agrees.

I still need to fix the CSS width problem.

### CODE REVIEW

**Comment by larsoner** [comment]
All better on the `_` names

https://7701-1301584-gh.circle-artifacts.com/0/html/generated/mne.preprocessing.ICA.html#mne.preprocessing.ICA

**Comment by agramfort** [comment]
good to go?

**Comment by larsoner** [comment]
Sure, I'll fix the col switch later

**Comment by cbrnr** [comment]
You fix stuff faster than I can even find the related issue/PR 😄 - however, don't auto-close #5118 just yet, I think we should address the references issue.

**Comment by larsoner** [comment]
The refs is almost certainly an upstream `numpydoc` bug, it takes care of refs.

**Comment by cbrnr** [comment]
OK, I might open an issue then. Unless of course you haven't already...

**Comment by larsoner** [comment]
Nope. You'll probably want to make a MWE first

**Comment by cbrnr** [comment]
Hm. I don't have numpydoc set up locally. Maybe I'll just ask and point them to our docs just to see if it isn't something obvious.

**Comment by larsoner** [comment]
Back to normal width (for better or worse):

https://7708-1301584-gh.circle-artifacts.com/0/html/python_reference.html

**Comment by cbrnr** [comment]
I like it better.

**Comment by larsoner** [comment]
For `numpydoc` you can `pip install numpydoc` then make a minimized version of our docs. If you don't have time I can do it

**Comment by cbrnr** [comment]
"make a minimized version of our docs" is what worries me... Please don't hesitate to do it, I think you will get it done much faster than me.

**Comment by larsoner** [comment]
For now I did this:

https://github.com/numpy/numpydoc/issues/169

I have a minimized version ready in case there is no good explanation

File: mne/preprocessing/xdawn.py [hunk 1]
```diff
@@ -364,15 +364,15 @@ class Xdawn(_XdawnTransformer):
...
+    patterns_ : dict of ndarray
         If fit, the Xdawn patterns used to restore the signals for each event
         type, else empty.
-    ``evokeds_`` : dict of evoked instance
+    evokeds_ : dict of evoked instance
         If fit, the evoked response for each event type.
-    ``event_id_`` : dict of event id
+    event_id_ : dict of event id
         The event id.
     ``correct_overlap_``: bool
```
Comment (cbrnr) [inline mne/preprocessing/xdawn.py]: Forgot this one.

Comment (cbrnr) [reply mne/preprocessing/xdawn.py]: (Probably because there's no space before the colon.)

File: mne/decoding/receptive_field.py [hunk 2]
```diff
@@ -55,16 +55,16 @@ class ReceptiveField(BaseEstimator):
...
-    ``coef_`` : array, shape ([n_outputs, ]n_features, n_delays)
+    coef_ : array, shape ([n_outputs, ]n_features, n_delays)
         The coefficients from the model fit, reshaped for easy visualization.
         During :meth:`mne.decoding.ReceptiveField.fit`, if ``y`` has one
         dimension (time), the ``n_outputs`` dimension here is omitted.
-    ``patterns_`` : array, shape ([n_outputs, ]n_features, n_delays)
+    patterns_ : array, shape ([n_outputs, ]n_features, n_delays)
         If fit, the inverted coefficients from the model.
-    ``delays_``: array, shape (n_delays,), dtype int
+    delays_: array, shape (n_delays,), dtype int
```
Comment (cbrnr) [inline mne/decoding/receptive_field.py]: Insert a space before the colon.

**Comment by cbrnr** [comment]
Other than my minor comments, LGTM!

**Comment by cbrnr** [comment]
Thanks @larsoner!

**Pull request merged**